guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collections.adoc

```diff
@@ -0,0 +1,102 @@
+[id="creating-a-playbook-with-modules-from-{project-context}-ansible-collections"]
+= Creating a playbook with modules from {Project} Ansible Collections
+
+Ansible modules from the {Project} Ansible Collections must be able to communicate with the {Project} API over HTTPS.
+In your playbooks, include parameters specifying how to authenticate to enable the connection to API.
+
+[IMPORTANT]
+====
+Do not store sensitive credentials, such as username and password, directly in your playbooks or environment variables.
+The following examples use Ansible vault to manage sensitive data.
+====
+
+.Prerequisites
+* A user account in {Project} exists with permissions to perform the action defined in your playbook.
+* You have access to one of the following types of authentication credentials for that user:
+** Username and password
+** Kerberos ticket, if your {ProjectServer} is configured to use {FreeIPA} as an external authentication source
```

Comment:
AD should work too

Reply:
We discussed this and agreed that even though AD should work too, we'll keep only a reference to FreeIPA here, among other things because we don't document Kerberos for Hammer authentication in the AD scenario either.